src/test/java/net/minecraftforge/debug/PotionEventTest.java

```diff
@@ -41,12 +41,6 @@
 
     public PotionEventTest()
     {
-    	FMLJavaModLoadingContext.get().getModEventBus().addListener(this::onSetup);
-    }
-    
-    private void onSetup(FMLCommonSetupEvent event)
-    {
-    	BrewingRecipeRegistry.addRecipe(Ingredient.of(Items.ICE), Ingredient.of(Items.LAVA_BUCKET), new ItemStack(Items.OBSIDIAN));
```

Comment:
I'm pretty sure this breaks the test mod.

Reply:
Well the method BrewingRecipeRegistry.addRecipe no longer exists so the call needs to be removed

Reply:
Then the test case itself should be added as a json.